version.adoc

```diff
@@ -0,0 +1 @@
+:current-version: 1.10
```

Comment:
Might it make sense to do this?:

`:current-version: 1.10 draft`

And then remove ` draft` pre-release?

Reply:
I had this locally at some point, but I removed it again because I felt that it looked odd in the `:Conventions` attribute both in the examples, which  would read
```
// global attributes:
  :Conventions = "CF-1.10 draft" ;
  :featureType = "timeSeries" ;
```
and in the text where this attribute is described, for example
```
[...] attribute **`Conventions`** to a string value that contains "**`CF-1.10 draft`**".
```
Hence my suggestion to add a second `:status-indicator:` or similar that can be combined with the `:current-version:` where appropriate.

But I am happy to use the simpler solution :)

Reply:
I don't have overly strong feelings here, but in the absence of Sturm und Drang from other corners my preference would be to go with using `current-version` for the whole shebang. Reasoning:
- It's simpler than having 2 attributes
- It doesn't result in invalid CDL, since space characters are allowed
and thus it would be easier to maintain, and also more obvious that one is working with a draft.